pkgs/by-name/sq/sqlitestudio/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  libsForQt5,
+  readline,
+  tcl,
+  python3,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+stdenv.mkDerivation rec {
+  pname = "sqlitestudio";
+  version = "3.4.4";
+
+  src = fetchFromGitHub {
+    owner = "pawelsalawa";
+    repo = "sqlitestudio";
+    rev = version;
+    hash = "sha256-5oBYv8WxyfVvvqr15XApvn6P/lBxR8b6E+2acRkvX0U=";
+  };
+
+  nativeBuildInputs =
+    with libsForQt5.qt5;
```

Comment:
If you want to let `copyDesktopItems` escape `with`, it should be put before `with`.